index.html

```diff
@@ -316,6 +315,80 @@
+          <span>Events</span>
+          <div class="underlines-container">
+            <svg width="4.5rem" height="0.2rem">
+              <rect
+                width="3.8rem"
+                height="0.2rem"
+                x="0.2rem"
+                rx="0.1rem"
+                fill="var(--color--events-title)"
+              />
+            </svg>
+            <svg width="0.5rem" height="0.2rem">
+              <rect width="0.5rem" height="0.2rem" rx="0.1rem" fill="var(--color--events-title)" />
+            </svg>
+          </div>
+        </h1>
+        <div class="events__cards-container container container_row">
+          <div class="events__card">
+            <h1 class="card__title card__title_external-event-bg">
+              <span>external</span>
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img
+                class="card-info__image"
+                src="./shared/events/usb-connection.png"
+                alt="connection"
+              />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
+                <p class="card-info__text_line">service for event organizers. Includes</p>
+                <p class="card-info__text_line">pricing, features, and types of accounts</p>
+              </div>
+              <a class="card-info__button card-info__button_success" href="#">Discover</a>
+            </div>
+          </div>
+          <div class="events__card">
+            <h1 class="card__title card__title_interactive-event-bg">
+              <span>interactive</span>
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img
+                class="card-info__image"
+                src="./shared/events/interactivity.png"
+                alt="connection"
+              />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
+                <p class="card-info__text_line">service for event organizers. Includes</p>
+                <p class="card-info__text_line">pricing, features, and types of accounts</p>
+              </div>
+              <a class="card-info__button card-info__button_danger" href="#">Discover</a>
+            </div>
+          </div>
+          <div class="events__card">
+            <h1 class="card__title card__title_internal-event-bg">
+              <span>internal</span>
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img class="card-info__image" src="./shared/events/hard-disk.png" alt="connection" />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
```

Comment:
You don't have to split the text in lines. Write it as is and limit it's width if necessary